hubblestack/files/hubblestack_nova/vulners_scanner.py

```diff
@@ -48,8 +48,8 @@ def audit(data_list, tags, labels, debug=False, **kwargs):
             vulners_data = _vulners_query(local_packages, os=os_name, version=os_version, api_key=data['vulners_api_key'])
             if 'result' in vulners_data and vulners_data['result'] == 'ERROR':
                 log.error(vulners_data['data']['error'])
-            vulners_data = _process_vulners(_vulners_query(local_packages, os=os_name, version=os_version))
```

Comment:
This looks like it was wrong all along. `_vulners_query()` is run twice, back to back, the second time failing to provide the `api_key`. Instead we'll cache the first results saved to `vulners_data` and use them again.